.golangci.yaml

```diff
@@ -77,8 +77,6 @@ linters-settings:
         alias: ctrl
       - pkg: k8s.io/apimachinery/pkg/util/runtime
         alias: utilruntime
-      - pkg: sigs.k8s.io/cluster-api/api/v1beta1
```

Comment:
why did you remove this? Looks valid!

Reply:
was throwing error in the CI, also idk why but I did run `make verify` before pushing the changes and it seems to be working fine but its failing in the CI

Reply:
Ran into some issues , fixing it

Reply:
Please use the following: 
```bash
$ make lint-golang-ci
docker run  --rm -t -i \
	-v /home/ria/go/pkg:/go/pkg \
	-v /home/ria/caph:/src/cluster-api-provider-hetzner \
	ghcr.io/syself/caph-builder:1.0.16 lint-golang-ci;
make: Entering directory '/src/cluster-api-provider-hetzner'
go version
go version go1.21.5 linux/amd64
golangci-lint version
golangci-lint has version 1.57.2 built with go1.22.1 from 77a8601a on 2024-03-28T19:01:11Z
golangci-lint run --out-format=github-actions
::error file=api/v1beta1/hcloudmachine_types.go,line=23,col=2::import "sigs.k8s.io/cluster-api/errors" imported without alias but must be with alias "capierrors" according to config (importas)
make: *** [Makefile:699: lint-golang-ci] Error 1
make: Leaving directory '/src/cluster-api-provider-hetzner'
make: *** [Makefile:692: lint-golang-ci] Error 2
```